src/dotnet/Chat/Emoji.cs

```diff
@@ -0,0 +1,17 @@
+namespace ActualChat.Chat;
+
+public record Emoji(string Code, string Name)
+{
+    private static readonly Dictionary<string, Emoji> _all = new Emoji[] {
+        new ("👍", "thumbs up"),
```

Comment:
I think we should add some negative ones too :)

Reply:
done